content/learn/book/welcome/why-bevy/_index.md

```diff
@@ -5,8 +5,71 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: discuss the cool features and design philosophy
+There are plently of great reasons to use Bevy, but here are just a few:
+
+### ECS 
+**TL;DR: Better workflow and abstraction in your code.**
+
+**LR:**
+
+Bevy has been designed to use a flexible data system in order to provide a smart, usable workflow. The engine uses an **Entity Component System** (ECS) to acheive this goal. In traditional game engines, a singular object (entity) contains its logic and components in one system. This is known as an object-oriented entity system.
+
+This has its benifits, however, in general, it is being phased out by ECSs. Instead of containing all the systems and components in a singular entity, in ECS, all of this is abstracted into structs (in Rust) and functions. Entities are defined as unique integers. 
+
+Components are structs that contain values for said components. Systems are Rust functions that are parsed into the main function and can specify components that an entity must have to be affected by the system.
+
+**This can allow for a great amount of abstraction that help with S.O.L.I.D principles.**
```

Comment:
This sentence doesn't add a ton of concrete value, and many readers won't know what S.O.L.I.D. is.

The overall idea of summarizing the section is good though.